graphene/types/field.py

```diff
@@ -40,11 +40,12 @@ class Person(ObjectType):
             last_name = graphene.Field(String, description='Surname')  # explicitly mounted as Field
 
     args:
-        type (class for a graphene.UnmountedType): must be a class (not an instance) of an
+        type (class for a graphene.UnmountedType): Must be a class (not an instance) of an
             unmounted graphene type (ex. scalar or object) which is used for the type of this
             field in the GraphQL schema.
-        args (optional, Dict[str, graphene.Argument]): arguments that can be input to the field.
-            Prefer to use **extra_args.
+        args (optional, Dict[str, graphene.Argument]): Arguments that can be input to the field.
+            Prefer to use ``**extra_args``, unless you use an argument name that clashes with one
+            of the Field arguments presented here (see :ref:`example<Scalars>`).
```

Comment:
`Scalars`: Don't you mean `ObjectType` where the example for using `args` is?

Reply:
Hmm yeah I was actually thinking the Scalars page had a nice list of the reserved but I think you're right: it makes sense to point back to the ObjectType page.